src/conn/conn_tiered.c

```diff
@@ -23,18 +23,46 @@
  * __flush_tier_once --
  *     Perform one iteration of tiered storage maintenance.
  */
-static void
+static int
 __flush_tier_once(WT_SESSION_IMPL *session, bool force)
 {
-    WT_UNUSED(session);
+    WT_CURSOR *cursor;
+    WT_DECL_RET;
+    const char *key, *value;
+
     WT_UNUSED(force);
+    __wt_verbose(session, WT_VERB_TIERED, "%s", "FLUSH_TIER_ONCE: Called");
     /*
      * - See if there is any "merging" work to do to prepare and create an object that is
      *   suitable for placing onto tiered storage.
      * - Do the work to create said objects.
      * - Move the objects.
      */
-    return;
+    cursor = NULL;
+    WT_RET(__wt_metadata_cursor(session, &cursor));
```

Comment:
Maybe instead of walking the metadata, this should walk the connection dhandle list.